rewrite-core/src/main/java/org/openrewrite/internal/ListUtils.java

```diff
@@ -162,17 +162,21 @@ private ListUtils() {
         for (int i = 0; i < ls.size(); i++) {
             int index = i;
 
-            ForkJoinTask<?> task = ForkJoinTask.adapt(() -> {
+            Runnable updateTreeFn = () -> {
                 T tree = ls.get(index);
                 T newTree = map.apply(index, tree);
                 if (newTree != tree) {
                     newLs.updateAndGet(l -> l == ls ? new ArrayList<>(ls) : l)
                             .set(index, newTree);
                 }
-            });
-
-            pool.invoke(task);
-            task.join();
+            };
+            if (null == pool) {
```

Comment:
```suggestion
            if (pool == null) {
```

Reply:
I've been putting null first in my null checks for so long now I'm not sure it'll be easy for me to change. :) Maybe I should write a recipe for that...